providers/sgx/sgx-app/src/main.rs

```diff
@@ -3,17 +3,27 @@ mod sgx_app;
 
 #[cfg(target_env = "sgx")]
 fn main() -> std::io::Result<()> {
-    let subscriber = tracing_subscriber::FmtSubscriber::builder()
-        .with_max_level(tracing::Level::INFO)
-        .finish();
-
-    tracing::subscriber::set_global_default(subscriber).expect("setting default subscriber failed");
     let mut args = std::env::args();
     let command = args.next();
     if command.is_none() {
         tracing::error!("no enclave command provided");
         return Err(std::io::ErrorKind::Other.into());
     }
+    let log_level = match args.next() {
```

Comment:
this extra arg would need to be passed in by SGX runner: https://github.com/crypto-com/tmkms-light/blob/main/providers/sgx/sgx-runner/src/command.rs#L35